runtime/onert/core/src/util/shapeinf/ReduceMin.cc

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "util/ShapeInference.h"
+
+namespace onert
+{
+namespace shape_inference
+{
+void StaticInferer::visit(const ir::operation::ReduceMin &op)
+{
+  const auto input_idx{op.getInputs().at(0)};
+  const auto &input = _operands.at(input_idx);
+
+  // get mutable output operand
+  const auto output_idx = op.getOutputs().at(0);
+  ir::Operand &output = _operands.at(output_idx);
+
+  // if input is dynamic, output also becomes dynamic
+  if (input.info().isDynamic())
+  {
+    output.info().setDynamic();
+    return;
+  }
+
+  const auto axes = op.param().axes;
+  const auto keep_dims = op.param().keep_dims;
+
+  // re-sizing output shape
+  ir::Shape new_shape = inferReduceShapes(input.info().shape(), axes, keep_dims);
+  output.info().shape(new_shape);
+}
+
+void DynamicInferer::visit(const ir::operation::ReduceMin &op)
+{
+  const auto input_idx{op.getInputs().at(0)};
+  const auto &input = _tensor_registry->getITensor(input_idx);
+  auto input_shape = getShape(input.get());
+
+  if (!input->is_dynamic())
+    return;
+
+  const auto axes = op.param().axes;
```

Comment:
`axis` can be nagetive number. Could you add applying it?

Reply:
The negative value of axis will handle in ```[ShapeInference.cc::inferRecudeShapes()](https://github.com/Samsung/ONE/blob/master/runtime/onert/core/src/util/shapeinf/ShapeInference.cc#L124)```.
My think, It is not problem.

Reply:
I haven't seen that part. Thanks.